pkg/adapter/bitbucket/adapter.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bitbucket
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudevents/sdk-go/pkg/cloudevents"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/client"
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/types"
+
+	"log"
+	"net/http"
+
+	sourcesv1alpha1 "github.com/knative/eventing-sources/pkg/apis/sources/v1alpha1"
+
+	webhooks "gopkg.in/go-playground/webhooks.v3"
+	bb "gopkg.in/go-playground/webhooks.v3/bitbucket"
+)
+
+const (
+	BBRequestUUID = "Request-UUID"
```

Comment:
I think  you want `bbRequestUUID` to not export the property.

Reply:
yes, you are right, thanks!